src/generic/srchctlg.cpp

```diff
@@ -474,15 +479,14 @@ void wxSearchCtrl::LayoutControls()
         return;
 
     const wxSize sizeTotal = GetClientSize();
-    int width = sizeTotal.x,
-        height = sizeTotal.y;
+    const int width = sizeTotal.x,
```

Comment:
I very much like making local variables const, but:

1. If width is const, so should be the height.
2. Please do it as a separate commit in the future.

Reply:
isn't this statement in C++ make them both const!?
`    const int width = sizeTotal.x,
              height = sizeTotal.y;`